### PR TITLE
(docs) update interface.md

### PR DESCRIPTION
**Pull request by vladdoster**
- correct spelling/grammar/punctuation
  * do not split infinitive
  * article agreement
- md header formatting consistency

### CODE REVIEW

File: docs/_docs/interface.md [hunk 1]
```diff
@@ -8,29 +8,28 @@ toc: false
 
 ## Home
 
-When you browse to the root of your registry, it's a fairly expected interface with
-links to collections and documentation. Here is what we see for the default "Tacosaurus Computing Center"
+When you browse to your registry's root, it's a reasonably expected interface with
```
Comment (vsoch) [inline docs/_docs/interface.md:11]: This reads strangely to me now - let's keep fairly, and also, periods and commas with quotes always go on the _inside_ so this is grammatically incorrect.

File: docs/_docs/interface.md [hunk 2]
```diff
@@ -8,29 +8,28 @@ toc: false
...
+links to collections and documentation. Here is what we see for the default "Tacosaurus Computing Center".
```
Comment (vsoch) [inline docs/_docs/interface.md:12]: This period should be on the inside of the quote.

File: docs/_docs/interface.md [hunk 3]
```diff
@@ -8,29 +8,28 @@ toc: false
...
-When you sign in with social authentication (e.g., GitHub) you are asked to sign standard terms of service.
-This basically says that the software is provided for you, free to use, and you are responsible
+When you sign in with social authentication (e.g., GitHub), you first must sign the standard terms of service (TOS).
+The TOS asserts that the software is provided for you, free to use, and you are responsible
 for taking care of your registry and are accountable for the containers within it.
 
 ![teams.png](../assets/img/interface/terms.png)
 
-When you agree, the robots welcome you!
+When you agree, robots welcome you!
```
Comment (vsoch) [inline docs/_docs/interface.md:22]: What is wrong with saying "the robots" ?

File: docs/_docs/interface.md [hunk 4]
```diff
@@ -8,29 +8,28 @@ toc: false
...
 
-
 ## Collections 
 
 A collection is a set of containers under the same namespace. For example, `dinosaur/avocado` and `dinosaur/banana` could be two containers in the `dinosaur` collection. You can browse all collections in the main collection view:
 
 ![collections.png](../assets/img/collections.png)
 
-and browse the containers within a collection by clicking on it. Here is a collection with one container:
+And browse the containers within a collection by clicking on it. Here is a collection with one container:
```
Comment (vsoch) [inline docs/_docs/interface.md:32]: This was a continuation of the previous sentence (hence lowercase)

**Review by vsoch — COMMENTED** [review]
Most of this is great! There are just a few changes that I'm not sure we should make, see my review comments.

File: docs/_docs/interface.md (hunk 1 above)
Comment (vladdoster) [reply docs/_docs/interface.md]: Doh! I need another espresso...

Fixed.

File: docs/_docs/interface.md (hunk 2 above)
Comment (vladdoster) [reply docs/_docs/interface.md]: make that a double shot

**Review by vsoch — APPROVED** [review]
This works for me :)

File: docs/_docs/interface.md (hunk 3 above)
Comment (vladdoster) [reply docs/_docs/interface.md]: It seemed unnecessary but now realize there are robots on the sregistry index URL. 

Reverted change.

File: docs/_docs/interface.md (hunk 4 above)
Comment (vladdoster) [reply docs/_docs/interface.md]: Fixed

**Review by vsoch — APPROVED** [review]
Thank you @vladdoster ! Here's some espressos to help :) :coffee: :coffee:

**Pull request merged**